deploy.sh

```diff
@@ -47,4 +47,7 @@ eval "${KUBECTL} create configmap es-config --from-file=es-config --dry-run -o y
 eval "${KUBECTL} create configmap fluentd-config --from-file=docker/fluentd/td-agent.conf --dry-run -o yaml" | eval "${KUBECTL} apply -f -"
 eval "${KUBECTL} create configmap kibana-config --from-file=kibana.yml --dry-run -o yaml" | eval "${KUBECTL} apply -f -"
 
+## Install RBAC policies
+eval "${KUBECTL} apply -f rbac"
+
```

Comment:
ES5 is ready to be used, but there is no proper webui yet.
I use it with kibana5 and x-pack so far.